src/main/java/org/generationcp/breeding/manager/listmanager/listcomponent/ListViewActionMenu.java

```diff
@@ -134,7 +134,7 @@ public void setActionMenuWhenListIsUnlocked(final boolean isLocalUserListOwner)
 		}
 	}
 
-	@PreAuthorize("hasAnyRole('ROLE_ADMIN') or hasRole('ROLE_SUPERADMIN')")
+	@PreAuthorize("hasAnyRole('ROLE_ADMIN')")
```

Comment:
Noted that Shiela has raised two tickets for adding granular permissions for this:
https://ibplatform.atlassian.net/browse/IBP-3008?oldIssueView=true
https://ibplatform.atlassian.net/browse/IBP-3007?oldIssueView=true